app/dashboard/models.py

```diff
@@ -1676,15 +1676,26 @@ def no_times_been_removed_by_staff(self):
             )
         return user_actions.count()
 
-    @property
-    def desc(self):
-        stats = self.stats
-        role = stats[0][0]
-        total_funded_participated = stats[1][0]
+    def get_desc(self, total_funded, total_fulfilled):
+        role = 'newbie'
+        if total_funded > total_fulfilled:
+            role = 'funder'
+        elif total_funded < total_fulfilled:
+            role = 'coder'
+
+        total_funded_participated = total_funded + total_fulfilled
         plural = 's' if total_funded_participated != 1 else ''
+
         return f"@{self.handle} is a {role} who has participated in {total_funded_participated} " \
                f"funded issue{plural} on Gitcoin"
 
+    @property
+    def desc(self):
+        total_funded = self.get_funded_bounties().count()
```

Comment:
@owocki @mbeacom 
I actually changed the behaviour now to use the fulfilled- vs funded bounties **count** not the usdt value difference between the two. Good or bad? 😅

Reply:
I did that as part of a little DRY'ing, though I later saw that it changed the behaviour and thought it might be actually a better measurement (Ha!)
Should I change it back or leave it as it is?

Reply:
i htink if it changes the behaviour it should be fixed or rolled back

Reply:
Done!